java/src/jmri/jmrix/openlcb/swing/ClientActions.java

```diff
@@ -35,24 +35,21 @@ public ClientActions(OlcbInterface iface) {
         this.iface = iface;
     }
 
-    public void openCdiWindow(NodeID destNode) {
+    public void openCdiWindow(NodeID destNode, String description) {
         final java.util.ArrayList<JButton> readList = new java.util.ArrayList<JButton>();
         final java.util.ArrayList<JButton> sensorButtonList = new java.util.ArrayList<JButton>();
         final java.util.ArrayList<JButton> turnoutButtonList = new java.util.ArrayList<JButton>();
 
         JmriJFrame f = new JmriJFrame();
-        f.setTitle("Configure " + destNode);
+        f.setTitle(Bundle.getMessage("CdiPanelConfigure") + " " + description);
```

Comment:
You do not want to build a string here, but want to pass everything to Bundle.getMessage(String, Object...) instead. The strings in .properties files should include placeholders for insertion following the rules for [MessageFormat](https://docs.oracle.com/javase/8/docs/api/java/text/MessageFormat.html).